maven/bnd-export-maven-plugin/src/test/resources/integration-test/test/export-from-dependencies/test.bndrun

```diff
@@ -1,4 +1,4 @@
--standalone: ${.}/index/index.xml.gz
+-standalone:
```

Comment:
I guess we're explicitly creating standalone bndruns so we don't rely on the `-standalone` directive to actually exist to enforce that so ... ok.